libs/tdb/files/tdb.cache.txt

```diff
@@ -0,0 +1,36 @@
+Checking simple C program: OK
```

Comment:
@hbl0307106015 No, you misunderstand my question.

With a `configure` script, if nominal parameters change in the environment, the script has a good probability of detecting those changes and adapting the settings.

Changes which are too large to adapt will cause either the `configure` script to fail, or else the compilation to fail.

The problem with statically configured settings is that there's no way to automatically validate that they still apply, or at least detect (through failure) when they don't as above.

Of course, a lot of packages build without autotools and work just fine, so maybe I'm just being overly cautious.